NewKernel_d/include/CGAL/NewKernel_d/Cartesian_base.h

```diff
@@ -22,6 +22,7 @@ namespace CGAL {
 template < typename FT_, typename Dim_, typename Derived_=Default>
 struct Cartesian_base_d : public CGAL_BASE
 {
+    typedef FT_  FT;
```

Comment:
Do you still need this, or was it solved by using Kernel_d_interface?

Reply:
right. just removed it again.